docs/templates/api_errors_page.md.j2

```diff
@@ -0,0 +1,7 @@
+| Category <td colspan=2> Code <td colspan=3>  Message |
+| -------- <td colspan=2> ---- <td colspan=3> -------- |
+{% for error_type, errors in error_types.items() %}
+{% for error in errors %}
+| {{ error_type }} <td colspan=2> {{ error['code'] }} <td colspan=3> {{ error['message'] }}|
+{% endfor %}
+{% endfor %}
```

Comment:
Please, add the empty line at the end of file. You can automatically [configure it](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file) in the IDE if you want.

Reply:
done